Week2/todo_app/program.js

```diff
@@ -21,39 +21,100 @@ class TodoModel {
         });
     }
 
-    create(description, callback) {
+    create(description, user_id, callback) {
         // Write code and query to create a new TODO item
+        let sqlQuery = "INSERT INTO  todo_items(text,is_completed,user_id) VALUES ('" + description + "'," + 0 + "," + user_id + ");"
```

Comment:
This technique works okay, but you could make it look a lot cleaner with the string templating syntax:
```
`VALUES ( "${ var }" )`
```

Reply:
This is also open to SQL injection, if SQL characters are injected into the values. I know it wasn't explained perfectly during the lecture, I'll work on some examples to illustrate what can happen. You could use escaping or prepared statements to address this.